includes/class-wc-stripe-account.php

```diff
@@ -99,7 +110,7 @@ public function get_cached_account_data( $mode = null ) {
 	 * @return array empty when no data found in transient, otherwise returns cached data
 	 */
 	private function read_account_from_cache( $mode = null ) {
```

Comment:
nit: `$mode` is no longer used. We can remove that parameter here and update line 97 above which appears to be the only place we call this function.